portfolio/src/main/webapp/style.css

```diff
@@ -0,0 +1,194 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  background-color: #192538;
+  font-family: "Montserrat", sans-serif;
+}
+
+li, a {
+  font-weight: 500;
+  font-size: 16px;
+  color: #edf0f1;
+  text-decoration: none;
+}
+
+nav {
+  
+  background-color: #192538;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 30px 10%;
+  position: fixed;
+  top: 0;
+  width: 100%;
+
+}
+
+
+.logo {
+  color: #edf0f1;
+  text-transform: uppercase;
+  letter-spacing: 5px;
+  font-size: 20px;
+  cursor: pointer;
+}
+
+.nav-links {
+  list-style: none; /* remove bullet points */
+}
+
+.nav-links li {
+  display: inline-block;
+  padding: 0px 20px;
+}
+
+
```

Comment:
nit:  extra space.  It looks harmless, but it would really help in the long run once you pickup the habit of keeping consistency even with minor things like this.

I mentioned on my earlier comment regarding trying out other IDEs that offer auto formatting.  I believe they also have tools to clean up extra spacing like these.